src/components-v2/common/dialogs/VaultDeposit.tsx

```diff
@@ -296,7 +300,11 @@ export const VaultDeposit = observer(({ open = false, vault, depositAdvisory, on
             </PercentagesContainer>
           </Grid>
         </Box>
-        <VaultFees vault={vault} className={classes.fees} onHelpClick={() => setShowFees(true)} />
+        {isInfluence && influenceVaultConfig !== undefined ? (
```

Comment:
should this throw an error? 
might be a good follow up - don't want to have partially defined influence data

Reply:
probably? would get a report either way, want me to add or follow up and merge this in?